.github/workflows/ui-tests.yml

```diff
@@ -0,0 +1,127 @@
+name: 'Run UI tests'
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build-summary:
+    runs-on: [ubuntu-latest]
+    name: Build summary UI tests
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Build junit plugin and download dependencies
+        run: mvn -V -ntp verify -Pskip --file plugin/pom.xml -Dgpg.skip
+      - name: Run UI tests for the tests summary on the build summary page of a job
+        env:
+          BROWSER: firefox-container
+        run: mvn -V -ntp test --file ui-tests/pom.xml -Dtest=BuildSummaryTest -Dgpg.skip -Dsurefire.rerunFailingTestsCount=1
+
+  build-results:
+    runs-on: [ubuntu-latest]
+    name: Build results UI tests
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
```

Comment:
could the version move to a variable?

Reply:
Interesting, normally dependabot updates those files as well. I have no idea why not in this place. I will replace that part with the config of the warnings plugin.